bin/ember

```diff
@@ -1,42 +1,6 @@
 #!/usr/bin/env node
 'use strict';
 
-// This file is intentionally left compatible with older versions of
-// node for the following scenario:
-// * user runs node 0.12
-// * user is still running an older node 0.12 compatible ember-cli locally
-// * user unintentionally installs ember-cli latest globally
+const loader = require('@std/esm')(module);
```

Comment:
We should move this out of `bin/ember` into `lib/cli/index.js`, and reset `bin/ember` to the prior state.  I don't see a reason to force this `bin/ember` incompatibility...